<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecDialog</class>
 <widget class="QDialog" name="SecDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>591</width>
     <height>241</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>show</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="update">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>220</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>update</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>140</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>190</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="tri">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>270</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>trier par capacite</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>270</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Numero</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>etat</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>type</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="combo">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>270</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Csearch">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>300</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pdf">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>300</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>pdf</string>
   </property>
  </widget>
  <widget class="QPushButton" name="print">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>print</string>
   </property>
  </widget>
  <widget class="QPushButton" name="video">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>videos </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>statistique</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
